src/main/java/seedu/address/model/person/Person.java

```diff
@@ -101,13 +98,13 @@ public boolean equals(Object other) {
                 && phone.equals(otherPerson.phone)
                 && email.equals(otherPerson.email)
                 && address.equals(otherPerson.address)
-                && tags.equals(otherPerson.tags);
+                && tier.equals(otherPerson.tier);
```

Comment:
Perhaps tiers not matching can still mean they are the same people, as perhaps they had a previous card tier already?